CMSgov/easi-app
# [EASi 3308] it gov admin actions emails

# EASI-3308

## Changes and Description

- Adds emails for the following actions:
  - Progress to new Step
  - Not Approved
  - Not an IT Gov Request
- Updates postman collection to add email recipients and add'l info where appropriate

## How to test this change

Run the automated tests, or:

1. Create an intake.
1. Close it with Not IT Gov/Not Approved or Progress it to a new step.
    - You can also remove optional fields like reason/feedback and add'l info where appropriate. 
1. Check mailcatcher.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
